backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/EvaluateExpressions.java

```diff
@@ -0,0 +1,76 @@
+package org.molgenis.emx2.sql;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.molgenis.emx2.Column;
+import org.molgenis.emx2.MolgenisException;
+import org.molgenis.emx2.Row;
+import org.molgenis.emx2.utils.TypeUtils;
+import org.molgenis.expression.Expressions;
+import scala.util.Try;
+
+public class EvaluateExpressions {
```

Comment:
@mswertz Is this file meant to test the expressions library? If so, shouldn't these tests be done inside of that repository?